app/controllers/OtherReasonController.scala

```diff
@@ -141,6 +145,7 @@ class OtherReasonController @Inject()(whenDidBecomeUnablePage: WhenDidBecomeUnab
       if(request.cookies.get("jsenabled").isDefined) {
         Future(Redirect(controllers.routes.OtherReasonController.onPageLoadForUploadEvidence(mode)))
       } else {
+        val nextPageWithUpload = navigation.nextPage(UploadFirstDocumentPage, mode)
```

Comment:
I don't think you need this, we have logic that will already redirect the user on success and failure